### PR TITLE
Scripts and Instructions for getting Baselines

### PR DESCRIPTION
**Pull request by mnltejaswini**
This PR includes Baseline HelloWorld App for Desktop CLR , scripts and instructions to get baseline numbers for HelloWorld MVC app for Desktop CLR and CoreCLR (standalone DNX)

### CODE REVIEW

File: testapp/SiteDensity/scripts/startDNXstandalonesites.ps1 [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Param(
+[string] $apppath,
+[string] $clr,
+[int32] $count
+
+)
+
+$scriptPath = 'D:\aspcore\startsite.ps1'
```
Comment (SajayAntony) [inline testapp/SiteDensity/scripts/startDNXstandalonesites.ps1]: Does this path block others from running this?

File: testapp/SiteDensity/src/Baseline_DesktopCLR_MVC4_HellowWorldApp/HelloWorldMVC/App_Start/RouteConfig.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿using System;
```
Comment (SajayAntony) [inline testapp/SiteDensity/src/Baseline_DesktopCLR_MVC4_HellowWorldApp/HelloWorldMVC/App_Start/RouteConfig.cs]: Need to add license headers.

**Comment by SajayAntony** [comment]
:watch:

**Comment by SajayAntony** [comment]
Do you want to merge this?

**Comment by SajayAntony** [comment]
ping

**Comment by DavidObando** [comment]
@mnltejaswini has this been merged?

**Pull request closed without merging**